chart/templates/deployment.yaml

```diff
@@ -108,11 +108,20 @@ spec:
         - name: CERT_PATH
           value: {{ .Values.activeMQ.cert_mount_path | quote }}
         - name: KEY_FILE_NAME
-          value: client_key
+          valueFrom:
```

Comment:
The KEY_FILE_NAME is a file name, not the file value.

If you update use valueFrom, this file name will become a long string value from the secret which will not work.